### ISSUE TITLE
generate_test_runner.rb ignores test function if // appears in line above within block comment

### ISSUE THREAD

**Issue by petermbauer** [ThrowTheSwitch/Unity#427]
The following minimal example shows that test functions with a block comment above it containing a `//` in the last line are silently ignored when generating the test runner source file, meaning there is no `RUN_TEST()` call for it generated in the `int main(void)` function of the respective `_Runner.c`.
```
/* // */
void test_something(void)
{
```

This may not be a common use-case but the issue also appears for e.g. URLs within the block comment:

```
/* http://some.domain.com */
void test_something(void)
{
```

In case this is relevant: this happens under Windows so with Windows line-endings.

**Comment by mvandervoord** [comment]
Added test to prove that this is not currently an issue. If you come across a test case where it is, please reopen. This should be in the next release.

**Comment by petermbauer** [comment]
Sorry, after digging into it again i found that the provided example was not complete. In addition to the `//` before the function there also needs to be at least one block comment in the test function body to trigger the issue.
From the following example test file
```
#include "unity.h"

/* See // */
void test_a(void)
{
    /* block comment */
}

/* See no slashes */
void test_b(void)
{
    /* block comment */
}

/* See // */
void test_c(void)
{
    // no block comment
}
```
only the `test_b` and `test_c` functions appear in the generated runner.c file:
```
...
/*=======MAIN=====*/
int main(void)
{
  suite_setup();
  UnityBegin("test.c");
  RUN_TEST(test_b, 10);
  RUN_TEST(test_c, 16);

  return suite_teardown(UnityEnd());
}
```

**Comment by mvandervoord** [comment]
Fixed with 2939c420ed9f2697bcb82c1b193bb48c4a8ad7a9
